texastribune/donations
# Add address lookup field

#### What's this PR do?
Adds address fields and an accompanying address lookup field to the main donation form.

#### Why are we doing this? How does it help us?
This is the front-end element of a larger project geared towards getting address information at time of donation. This will keep membership from having to do extra work when wanting to send something to a member via the mail. Also, getting address information at time of donation is fairly common.

#### How should this be manually tested?
Locally or on staging...
* on the main donation form, find the address lookup field halfway down the form
* enter an address and then select the desired address from the drop-down
* ensure that the address pieces populate the address, city, state and zip code fields appropriately

#### How should this change be communicated to end users?
We're going to let the membership team play around with these updates as well to give us feedback on layout/design.

#### Are there any smells or added technical debt to note?
Not really a smell, but worth mentioning that we added a new front-end library called "vue-google-autocomplete" to simplify some of the lookup implementation.

#### What are the relevant tickets?
https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwS1XPty68eK4Ett/rec2Gm8BTDPBefvFI?blocks=hide

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
